tracing/http.go

```diff
@@ -0,0 +1,228 @@
+package tracing
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"go.opencensus.io/plugin/ochttp"
+	"go.opencensus.io/trace"
+)
+
+const (
+	//RequestHeaderAnnotationPrefix is a prefix which added to each request header attribute
+	RequestHeaderAnnotationPrefix = "request.header."
+
+	//ResponseHeaderAnnotationPrefix is a prefix which added to each response header attribute
+	ResponseHeaderAnnotationPrefix = "response.header."
+
+	//RequestPayloadAnnotationKey is a key under which request payload stored in span
+	RequestPayloadAnnotationKey = "request.payload"
+
+	//ResponsePayloadAnnotationKey is a key under which response payload stored in span
+	ResponsePayloadAnnotationKey = "response.payload"
+
+	//DefaultMaxPayloadSize represent max payload size which will be added to span
+	DefaultMaxPayloadSize = 1024 * 1024
+
+	//ReducedMarkerKey is a key for annotation which will be presented in span in case payload was reduced
+	ReducedMarkerKey = "reduced_payload"
+
+	//ReducedMarkerValue is a value for annotation which will be presented in span in case payload was reduced
+	ReducedMarkerValue = "true"
```

Comment:
is it a good idea to use "true" string here? It is something we invented or values is forced by 3rd party?

Reply:
It our own marker which we add to span with reduced body, to allow developers to filter out them, if they want to see spans only with full body, for me "true" good, any idea how to do it more descriptive ?

Reply:
I was thinking about using "reduced" or "enabled" string instead of "true". I am fine with any option (including existing), so it is up to you.

Reply:
Actually we can discuss it, is it usefull to add some marker that span contains payload and signal not only that payload reduced but signal that payload actually exists for example payload.type = reduced|enabled allow developers to have more granural filtering, we will think about it

Reply:
Are we compressing the payload ("reduced"), truncating (simply cut off after X characters and inject an ellipse to visually indicate truncation), or replacing the data (replacing data with a marker indicating that data has been replaced)

Reply:
That a good point @mphillips-infoblox, i will change reduced to truncated actually, I already put "..." in the body but label for me seems reasonable to add annotation as well, in case somebody for example what to filterout traces without full body

Reply:
Yes, as @Evgeniy-L suggested, if you are consistent and use something like payload.truncated = true, that would work really well